index.html

```diff
@@ -142,58 +142,52 @@ <h2>About <br> Myself</h2>
           <li><a href="#"><img src="images/M.jpg" alt="M"></a></li>
         </ul>
         <nav>
-          <button type="button" class="button">Get my resume</button>
+          <button type="button" class="button-2">Get my resume</button>
         </nav>
       </div>
+      </div>  
+      
+      <section class="others">
+          <article class="my-works" id="my-works-2">
+            <h3>Languages</h3><i class="fa-solid fa-angle-down"></i>
+          </article>
+            <ul class="lang">
+              <li><img src="images/javascript logo.jpg" alt="javascript"><p>Javascript</p></li>
+              <li><img src="images/HTML logo.jpg" alt="HTML"><p>HTML</p></li>
+              <li><img src="images/css logo.jpg" alt="css"><p>CSS</p></li>
+            </ul>
+          <article class="my-works">
+            <h3>Frameworks</h3><i class="fa-solid fa-angle-right"></i>
+          </article>
+          <article class="my-works">
+            <h3>Skills</h3><i class="fa-solid fa-angle-right"></i>
+          </article>
+      </section>
+    </section>
+  </main>
 
-      <div class="others">
-        <ul>
-          <li>Languages<a href="#"><span><img src="images/Union.jpg"></a></span>
-            <div id="lang">
-              <a href="#"><img src="images/Langguage List (1).jpg" width="270"></a>
-            </div>
-          </li>
-          <li>Frameworks<a href="#"><span><img src="images/Union 2.jpg"></span></a>
-            <div>
-              <a href="#"><img src="images/Separator Bottom.jpg" width="270"></a>
-            </div>
-          </li>
-          <li>Skills<a href="#"><span id="two"><img src="images/Union 2.jpg"></span></a>
-            <div>
-              <a href="#"><img src="images/Separator Bottom.jpg" width="270"></a>
-            </div>
-          </li>
-        </ul>
-      </div>
+  <section class="formsection">
+    <div class="heading-2">
+      <h2>Contact me</h2>
```

Comment:
- [ ] [OPTIONAL] Great job adding the background image as requested by the previous reviewer 🐙. Please, you said you are using animation to center the form, but I think it is not working, Kindly, check and fix it. 🥇

![image](https://user-images.githubusercontent.com/91301423/201475479-d6fc0de5-d6aa-4ee2-a578-0000968b28a1.png)